UnitsNet/Scripts/UnitDefinitions/ForcePerLength.json

```diff
@@ -0,0 +1,35 @@
+﻿{
+  "Name": "ForcePerLength",
+  "BaseUnit": "NewtonPerMeter",
+  "XmlDoc": "Force per length is the amount of change applied per length.",
```

Comment:
Not sure this XmlDoc sentence make sense to me:

> amount of change applied per length

In my mind it would read something like:

> The magnitude of force per unit length.

Normally we copy some summary from wiki, but I don't see any good definitions when googling `force per length`, but I found this https://en.wikipedia.org/wiki/Amp%C3%A8re's_force_law .